base/compiler/types.jl

```diff
@@ -47,14 +47,20 @@ MethodInfo(src::CodeInfo) = MethodInfo(
 A special wrapper that represents a local variable of a method being analyzed.
 This does not participate in the native type system nor the inference lattice, and it thus
 should be always unwrapped to `v.typ` when performing any type or lattice operations on it.
+
 `v.undef` represents undefined-ness of this static parameter. If `true`, it means that the
 variable _may_ be undefined at runtime, otherwise it is guaranteed to be defined.
 If `v.typ === Bottom` it means that the variable is strictly undefined.
+
+`v.ssadef` represents the "reaching definition" for the variable. If negative, this refers
+to a "virtual ϕ-block" preceding the given index. If a slot has the same `ssadef` at two
```

Comment:
This is the same meaning as in `stupdate!`, but it doesn't mean "we can't track the def"

An ssadef of `-pc` means "If this code were in SSA form, a use of this slot would be replaced with a use of a `ϕ-node`." Which ϕ-node? The one that would have been inserted at `pc`

That's important because the rule given above:
> If the reaching def is equal at two program points, then the slot contents are guaranteed to be egal (i.e. x₀ === x₁)

Also works for `-pc` ssadefs